docs/PODMAN_INSTALL_TUNING.md

```
@@ -0,0 +1,139 @@
+# Swappiness and I/O tuning
+vm.swappiness = 1
+vm.overcommit_memory = 1
+
+# Network connection backlogs
+net.core.somaxconn = 65535
+net.core.netdev_max_backlog = 32768
+net.ipv4.tcp_max_syn_backlog = 32768
+```
+
+Apply settings live:
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)